byway/src/main/webapp/home.js

```diff
@@ -6,11 +17,107 @@ else{
 }
 
 function initializeHomePage(){
-  // TODO(renau06): fetch /api/gettrips and display the users past trips
+  setLogoutLink();
+  createPastTrip();
   document.getElementById('create-trip').addEventListener('click', () => {
     fetch('/api/createtrip', {method: 'POST'}).then((response) => response.json()).then((trip) =>{
-      window.location.href = '/destinations.html?tripKey=' + trip.keyString;  
+      let tripKey = trip.keyString;
+      window.location.href = configureTripKeyForPath(tripKey, "/destinations.html");
     });
   });
 }
 
+function createPastTrip(){
```

Comment:
Nit: it may be more accurate to call this `loadPastTrips` or something along those lines. "createPastTrip" is a bit confusing, it sounds like you are creating new trips from past info something. This doesn't really create trips, so the "create" part is not strictly accurate.

Reply:
changed to load